Library/.rubocop.yml

```diff
@@ -1,5 +1,7 @@
 AllCops:
   TargetRubyVersion: 2.0
+  Exclude:
+    - '**/Casks/**/*'
```

Comment:
The `rubocop-cask` gem includes a default RuboCop configuration for Casks that is quite different from the one for brew core: https://github.com/caskroom/rubocop-cask/blob/master/config/default.yml